packages/dataviews/src/dataviews.js

```diff
@@ -37,6 +37,7 @@ export default function DataViews( {
 	deferredRendering = false,
 } ) {
 	const [ selection, setSelection ] = useState( [] );
+	const [ openFilterOnMount, setOpenFilterOnMount ] = useState( null );
```

Comment:
What do you think of renaming this to something like `openedFilter, setOpenedFilter`?